Tenant.rb

```diff
@@ -0,0 +1,3 @@
+class Tenant < ActiveRecord::Base
```

Comment:
This all looks good, except again, note that the file name should be lower case.